docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/drift-detection.adoc

```diff
@@ -271,3 +271,4 @@ Listed here are causes that may prevent drift detection.
 ** `Traced Azure resources are manually modified`
 ** `Traced GCP resources are manually modified`
 
+* Please note that at this time, AWS Route53 is not supported for tracing and drift detection.
```

Comment:
Option 1: NOTE: Tracing and drift detection functionalities are not presently supported for AWS Route53 resources.

Option2: NOTE: AWS Route53 is currently not available for tracing and drift detection.